autoendpoint/src/error.rs

```diff
@@ -152,6 +152,11 @@ impl ApiErrorKind {
         }
     }
 
+    /// Should the User record be removed?
+    pub fn user_still_valid(&self) -> bool {
```

Comment:
wanna kill this?

Reply:
crap, must have snuck in during the refactor.